components/arena/python/arena/_arena_standalone_op.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import kfp.dsl as dsl
+import datetime
+import logging
+
+
+# def arena_submit_standalone_job_op(name, image, gpus: int, ):
+
+class StandaloneOp(dsl.ContainerOp):
+  """Submit standalone Job."""
```

Comment:
Can you add more comprehensive docstring?

Reply:
Sure, I will update.